File: 01. SourceCode/ChildFund/ChildProfile.Model/Model/LoginProfileModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ChildProfiles.Model
{
    public class LoginProfileModel
    {
        /// <summary>
        /// Id user
        /// </summary>
        public string Id { get; set; }
        /// <summary>
        /// Tên đăng nhập
        /// </summary>
        public string Name { get; set; }
        /// <summary>
        /// Tài khoản đăng nhập
        /// </summary>
        public string UserName { get; set; }
        /// <summary>
        /// Cấp sử dụng tài khoản
        /// </summary>
        public string UserLever { get; set; }
        /// <summary>
        /// Id vùng hoạt động
        /// </summary>
        public string AreaUserId { get; set; }

        public string ProvinceId { get; set; }
        public string DistrictId { get; set; }
        public string WardId { get; set; }
        /// <summary>
        /// Trạng thái hoạt động tài khoản
        /// </summary>
        public bool IsDisable { get; set; }
        /// <summary>
        /// Danh sách quyền
        /// </summary>
     //   public List<string> ListRoles { get; set; }
        /// <summary>
        /// Image path
        /// </summary>
        public string ImagePath { get; set; }
        public string SecurityKey { get; set; }
    }
}

File: 01. SourceCode/Android/ChildProfile/app/src/main/java/nts/childprofile/model/LoginProfileModel.java
package nts.childprofile.model;

import java.util.List;

/**
 * Created by NTS-VANVV on 29/12/2018.
 */

public class LoginProfileModel {
    /// <summary>
    /// Id user
    /// </summary>
    public String Id;

    /// <summary>
    /// Tên đăng nhập
    /// </summary>
    public String Name;

    /// <summary>
    /// Tài khoản đăng nhập
    /// </summary>
    public String UserName;

    /// <summary>
    /// Cấp sử dụng tài khoản
    /// </summary>
    public String UserLever;

    /// <summary>redi
    /// Id vùng hoạt động
    /// </summary>
    public String AreaUserId;

    /// <summary>
    /// Id tỉnh hoạt động
    /// </summary>
    public String ProvinceId;

    /// <summary>
    /// Id huyện hoạt động
    /// </summary>
    public String DistrictId;

    public String WardId;

    /// <summary>
    /// Trạng thái hoạt động tài khoản
    /// </summary>
    public Boolean IsDisable;

    /// <summary>
    /// Danh sách quyền
    /// </summary>
    public List<String> ListRoles;

    /// <summary>
    /// Image path
    /// </summary>
    public String ImagePath;
}
